webapp/portlet/src/main/webapp/activity-composer-app/components/ExoActivityComposer.vue

```diff
@@ -23,6 +23,24 @@
               <i class="uiIconError"></i>{{ $t('activity.composer.post.error') }}
             </div>
           </transition>
+          <div v-if="attachments.length" class="attachments-list">
+            <i class="uiIconAttach"></i>
+            <p class="attachedFiles">{{ $t('attachments.drawer.title') }} ({{ attachments.length }})</p>
+          </div>
+          <div class="composerApps">
```

Comment:
I would call them `composerActions` instead ? It is not necessary an app. WDYT ?

Reply:
Yes, it's more clear!